docs/monitoring/monitoring.md

```diff
@@ -114,6 +114,152 @@ spec.values.prometheus-node-exporter.resources.requests.memory: 30Mi
+
+```
+cat << EOF > a-single-receiver.yaml
+apiVersion: monitoring.coreos.com/v1alpha1
+kind: AlertmanagerConfig
+metadata:
+  name: amc-example
+  # namespace: your value
+  labels:
+    alertmanagerConfig: example
+spec:
+  route:
+    continue: true
+    groupBy:
+    - cluster
+    - alertname
+    receiver: "amc-webhook-reciver"
+  receivers:
+  - name: "amc-webhook-reciver"
+    webhookConfigs:
+    - sendResolved: true
+      url: "http://192.168.122.159:8090/"
+EOF
+
+# kubectl apply -f a-single-receiver.yaml
+alertmanagerconfig.monitoring.coreos.com/amc-example created
+
+# kubectl get alertmanagerconfig -A
+NAMESPACE   NAME          AGE
+default     amc-example   27s
+
+```
+
+#### Example of webhook received alert
+
+The alert sent to an webhook server will be in following format:
+
+```
+{
+'receiver': 'longhorn-system-amc-example-amc-webhook-reciver', 
+'status': 'firing', 
+'alerts': [], 
+'groupLabels': {}, 
+'commonLabels': {'alertname': 'LonghornVolumeStatusWarning', 'container': 'longhorn-manager', 'endpoint': 'manager', 'instance': '10.52.0.83:9500', 'issue': 'Longhorn volume is Degraded.',
+'job': 'longhorn-backend', 'namespace': 'longhorn-system', 'node': 'harv2', 'pod': 'longhorn-manager-r5bgm', 'prometheus': 'cattle-monitoring-system/rancher-monitoring-prometheus',
+'service': 'longhorn-backend', 'severity': 'warning'},
+'commonAnnotations': {'description': 'Longhorn volume is Degraded for more than 5 minutes.', 'runbook_url': 'https://longhorn.io/docs/1.3.0/monitoring/metrics/',
+'summary': 'Longhorn volume is Degraded'},
+'externalURL': 'https://192.168.122.200/api/v1/namespaces/cattle-monitoring-system/services/http:rancher-monitoring-alertmanager:9093/proxy',
+'version': '4',
+'groupKey': '{}/{namespace="longhorn-system"}:{}',
+'truncatedAlerts': 0
+}
+```
+
+:::note
+
+Different receiver may represent the alert in it's format, please refer the related documents.
+
+:::
+
+#### Known limitation
+
+The `AlertmanagerConfig` is enforced by namespace. Gloabl level `AlertmanagerConfig` without namespace is not supported.
+
+One github issues is created to track upstream change, once the feature is available, `Harvester` will adapt it.
+
+https://github.com/harvester/harvester/issues/2760
+
+### View and manage alerts
+
+
+#### From Alertmanager dashboard
+
+From below link, user may visit the `Alertmanager` original dashboard.
+
+> https://the-cluster-vip/api/v1/namespaces/cattle-monitoring-system/services/http:rancher-monitoring-alertmanager:9093/proxy/#/alerts
+
+Overall view of `Alertmanager` dashboard.
+
+![](/img/v1.1/monitoring/alertmanager-dashboard.png)
+
+View the details of an alert.
+
+![](/img/v1.1/monitoring/alert-view-detail.png)
+
+#### From Prometheus dashboard
+
+From below link, user may visit the `Prometheus` original dashboard, which has detailed information of alert.
+
+> https://the-cluster-vip/api/v1/namespaces/cattle-monitoring-system/services/http:rancher-monitoring-prometheus:9090/proxy/
```

Comment:
```suggestion
You can visit the original dashboard of `Prometheus` from [here](https://the-cluster-vip/api/v1/namespaces/cattle-monitoring-system/services/http:rancher-monitoring-prometheus:9090/proxy/), which offers detailed information about the alert.
```

Reply:
Similar. Because the link itself is invalid, user need to copy the link and change the `the-cluser-vip` to real vip address (e.g. 192.168.122.100). Here we need to represent the raw link text for user to copy and modify.